3rdparty/openssl/CMakeLists.txt

```diff
@@ -63,11 +64,12 @@ list(
   APPEND
   OPENSSL_3_ASSEMBLY_SCRIPTS
   ${OPENSSL_ASSEMBLY_SCRIPTS}
+  crypto/x86_64cpuid
+  crypto/aes/asm/aesni-x86_64
+  crypto/aes/asm/vpaes-x86_64
+  crypto/sha/asm/sha256-x86_64
   crypto/aes/asm/aes-x86_64
-  crypto/aes/asm/bsaes-x86_64
```

Comment:
why were these removed

Reply:
They were added back later in the script. I'm using this file list exclusively for assembly files that have been processed by Intel. These are not.